plugins/cfhour.js

```diff
@@ -85,9 +87,9 @@ module.exports = (function(){
 				}
 				else {
 					console.error(error);
+					onErrorDebounce(error);
```

Comment:
Yes youre right.  Eventually it might be nice to provide an onError on the bot itself that is debounced and make the label defaulted to blank or something.